apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/ErrorReporting/ReportEventsTo.cs

```diff
@@ -22,7 +22,7 @@ namespace Google.Cloud.Diagnostics.Common
     /// <summary>
     /// The location error events will be sent to.
     /// </summary>
-    public enum ReportEventsToLocation
+    public enum EventTarget
```

Comment:
`ReportEventsTo.Logging` sounds imperative though - whereas really the result needs to be used, doesn't it?

```csharp
// Looks okay, but is a bug, because the return value isn't used
ReportEventsTo.Logging(...);

// Correct, but having a value of type "ReportEventsTo" seems weird
var reportEventsTo = ReportEventsTo.Logging(...);
```

If we had `EventTarget` you could have:

```csharp
// Sounds wrong - which is great, because it *is* wrong
EventTarget.ForLogging(...);

// Looks right and is right
var target = EventTarget.ForLogging(...);
```

But maybe I'm missing something?